OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -100,8 +105,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                     </list>
                                 </collation>
+                                <condition key="deficient">
+                                  The manuscript is in bad condition. The upper board is broken in two halves and the outer is missing, together with the lower board.
```

Comment:
Do "upper" and "lower" refer to the front and back board? This terminology is a little confusing to me – is this standard?

Reply:
Upper and front or lower and back are synonyms, I think I used upper and lower because they were used in the other project I worked. I need to go through all the descriptions and check it! (actually it would be better to talk about right and left board, to avoid any confusion)

Reply:
My feeling is that front and back board are used most commonly in Ethiopian studies, and since they are not wrong, I would suggest to stick to them.